clang-tools-extra/clang-tidy/modernize/MinMaxUseInitializerListCheck.h

```diff
@@ -0,0 +1,65 @@
+//===--- MinMaxUseInitializerListCheck.h - clang-tidy -----------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_CLANG_TOOLS_EXTRA_CLANG_TIDY_MODERNIZE_MINMAXUSEINITIALIZERLISTCHECK_H
+#define LLVM_CLANG_TOOLS_EXTRA_CLANG_TIDY_MODERNIZE_MINMAXUSEINITIALIZERLISTCHECK_H
+
+#include "../ClangTidyCheck.h"
+#include "../utils/IncludeInserter.h"
+
+namespace clang::tidy::modernize {
+
+/// Replaces chained ``std::min`` and ``std::max`` calls with a initializer list
```

Comment:
`chained` sound like statement after statement. I think `nested` sounds better, wdyt?
Also: `an initializer list` instead of `a initializer list` (same in release notes and check docs).

Of course, chained `min`/`max` calls could also be detected, but that may be better left for a follow-up pr.